src/main/java/global/banking/Algo1.java

```diff
@@ -0,0 +1,29 @@
+package global.banking;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Algo1 {
+    /**
+     * To have O(n) complexity we will use HashMap
+     */
+
+    public static void main(String[] args) {
```

Comment:
encapsulate the implemetation to its own method and return an array of 2 integers with the 2 positions